### PR TITLE
[MIRROR] Ports a Boatload of Teshari Customization from CitRP, Chomp & YW, Small Update to Protogen Markings

### PR DESCRIPTION
**Pull request by CHOMPStationBot**
Original PR: https://github.com/VOREStation/VOREStation/pull/9312
--------------------
All changes to the standard non _vr eyes, gloves, suit, undercoat/deptuniform and ties.dmi are due to be ported upstream to Polaris
The orange goggles are going to be ported to Polaris

This PR will be referred to from Polaris repo re: the assets going their way then this will be marked ready for review

Changes:
Adds Teshari-fitted collar sprites (with refitting logic from YawnWider).
Adds Teshari-fitted sleek orange goggles (from CitRP).
Adds Teshari-fitted rings (from CitRP).
Adds better Teshari-fitted Space Suit & Helmet sprites (by KBraid).
Adds better Teshari-fitted Radiation Suit & Helmet sprites (by KBraid).
Adds full sets of Teshari standard & job cloaks from CitRP (different sprites than Polaris) - job cloaks can be selected as an option when selected in loadout. Captain, Psych, RD, HoS & HoP cloaks are net-new (from CitRP).
Adds the dark retrowave & black + glowing Teshari cloaks (from Chomp).
Adds full sets of Teshari standard & job belted cloaks (from Chomp).
Adds full set of Teshari belted cloaks (from Chomp).
Adds better Teshari standard & job undercoat sprites (from CitRP).
Further enhances colouration of protogen markings.

### CODE REVIEW

**Comment by KasparoVy** [comment]
This was a backport of a number of stuff from you kind folks

**Comment by KasparoVy** [comment]
Cheers! You folks had a bunch of awesome sprites.

File: code/modules/clothing/under/accessories/accessory_vr.dm [hunk 1]
```diff
@@ -12,6 +12,37 @@
...
+//Forces different sprite sheet on equip
+/obj/item/clothing/accessory/choker/New()
+	..()
+	icon_previous_override = icon_override
+
+/obj/item/clothing/accessory/choker/equipped() //Solution for race-specific sprites for an accessory which is also a suit. Suit icons break if you don't use icon override which then also overrides race-specific sprites.
+	..()
+	setUniqueSpeciesSprite()
+
+/obj/item/clothing/accessory/choker/proc/setUniqueSpeciesSprite()
```
Comment (KasparoVy) [inline code/modules/clothing/under/accessories/accessory_vr.dm:29]: Just tried to make this refitting business a bit more flexible so it ain't just hardcoded refit for Tesh only

Comment (cadyn) [reply code/modules/clothing/under/accessories/accessory_vr.dm:29]: bet. This is just annoying to deal with and I'm way too tired and not even supposed to be doing dev work on the weekends for sanity.

Comment (cadyn) [reply code/modules/clothing/under/accessories/accessory_vr.dm:29]: when you are you guys gonna backport my spicy ballistics code? :>

Comment (KasparoVy) [reply code/modules/clothing/under/accessories/accessory_vr.dm:29]: @cadyn No clue, would be great to have but I'm almost solely focused on aesthetics and improving character-building

Comment (cadyn) [reply code/modules/clothing/under/accessories/accessory_vr.dm:29]: Fair enough, pff, anyways thanks and you too <3

**Pull request merged**